documentation/docs/mock-apps/services/process-digital-conformity-credential.md

```diff
@@ -0,0 +1,88 @@
+---
+sidebar_position: 53
+title: Process Digital Conformity Credential
+---
+
+import Disclaimer from '../../\_disclaimer.mdx';
+
+<Disclaimer />
+
+## Description
+
+The `processDigitalConformityCredential` service is responsible for processing a digital conformity credential, issuing a [Verifiable Credential (VC)](https://uncefact.github.io/spec-untp/docs/specification/VerifiableCredentials), uploading it to the [Storage service](/docs/mock-apps/dependent-services/storage-service), registering the link to the stored digital conformity credential with the [Identity Resolver service](/docs/mock-apps/dependent-services/identity-resolution-service). It handles the entire lifecycle of creating and managing a digital conformity credential, from data input to storage and resolution.
+
+## Diagram
+
+```mermaid
+sequenceDiagram
+participant C as Client
+participant P as processDigitalConformityCredential
+participant V as VCKit
+participant S as Storage
+participant D as DLR
+C->>P: Call processDigitalConformityCredential(digitalConformityCredential, context)
+P->>P: Validate context
+P->>P: Extract identifier
+P->>V: Issue VC
+V-->>P: Return VC
+P->>S: Upload VC
+S-->>P: Return VC URL
+P->>D: Register link resolver
+D-->>P: Return resolver URL
+P-->>C: Return digital conformity credential VC and resolver URL
+```
+
+## Example
+
+```json
+{
+  "name": "processDigitalConformityCredential",
+  "parameters": [
+    {
+      "vckit": {
+        "vckitAPIUrl": "https://api.vckit.example.com",
+        "issuer": "did:example:123456789abcdefghi"
+      },
+      "digitalConformityCredential": {
+        "context": [
+          "https://jargon.sh/user/unece/ConformityCredential/v/0.5.0/artefacts/jsonldContexts/ConformityCredential.jsonld?class=ConformityCredential"
+        ],
+        "type": ["VerifiableCredential", "DigitalConformityCredential"],
```

Comment:
Should we delete `VerifiableCredential` in type, it setup as default when we issue a credential